unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/WebInterfacePassportApiBridge.cs

```diff
@@ -21,6 +19,15 @@ await DCL.Environment.i.platform.serviceProviders.theGraph.QueryNftCollections(u
             return nftList;
         }
 
+        public async UniTask<Nft> QueryNftCollectionEthereum(string userId, string urn)
```

Comment:
This needs to support cancellation

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/4006/commits/b2664bebab0dbfe34c26f10fa95feba65e77b41b. Thanks!!